packages/wallets/src/features/cfd/screens/CreatePasswordMT5/CreatePasswordMT5.scss

```diff
@@ -0,0 +1,52 @@
+.wallets-create-password-mt5 {
+    width: 45.5rem;
+    border-radius: 0.8rem;
+    background: var(--system-light-8-primary-background, #fff);
+
+    &__text {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        gap: 0.8rem;
+
+        @include mobile {
```

Comment:
use the latest breakpoints provided `mobile-screen` `mobile-or-tablet-screen`, `desktop-screen`.